src/index.js

```diff
@@ -8,9 +8,9 @@ import 'bootstrap/dist/css/bootstrap.css'
 
 const root = ReactDOM.createRoot(document.getElementById('root'));
 root.render(
-  <React.StrictMode>
```

Comment:
Please add this tag back. This is important tag to let us find the bugs in our code